doc: add an example to explain the diff (#31970)
kubernetes/website: content/en/docs/tasks/debug-application-cluster/debug-running-pod.md

/ #
```

> You can view a newly created pod `myapp-debug` with using `kubectl get`:

Reviewer: ```suggestion
Using `kubectl get pod`, you can see a new pod `myapp-debug` has been created:
```

Reviewer 2: +1 to @tengqm 's suggestion
